collections/_news/2021-11-05-coindesk.md

```diff
@@ -0,0 +1,9 @@
+---
+title:  "Chia Network to Develop Prototype for World Bank’s Climate Warehouse"
+weblink: "https://www.coindesk.com/business/2021/11/05/chia-network-to-develop-prototype-for-world-banks-climate-warehouse/"
+date:   2021-11-04
```

Comment:
The correct date is 2021-11-05 but then it comes before the Bloomberg article on the sort in the news page.